### ISSUE TITLE
'ital' axis value in STAT table should be a boolean

### ISSUE THREAD

**Issue by RosaWagner** [googlefonts/fontbakery#3668]
### `Ital` axis value in italic font should be `0` and elided in Roman, ans `1` and not elided in Italic, value `0` should be linked to `1`

https://docs.microsoft.com/en-us/typography/opentype/spec/dvaraxistag_ital#axis-definition
It often happens that designers assign the italic angle as a value to the Italic instance in the STAT table. So it can't be the universal profile. But it goes against recommendation of the opentype spec, and forbidden by google fonts so far.

**In Italic VF**
```
      </AxisValue>
      <AxisValue index="11" Format="1">
        <AxisIndex value="2"/>
        <Flags value="0"/>
        <ValueNameID value="261"/>  <!-- Italic -->
        <Value value="1.0"/>
      </AxisValue>
    </AxisValueArray>

```

**In Roman VF**
```
</AxisValue>
      <AxisValue index="11" Format="3">
        <AxisIndex value="2"/>
        <Flags value="2"/>  <!-- ElidableAxisValueName -->
        <ValueNameID value="279"/>  <!-- Roman -->
        <Value value="0.0"/>
        <LinkedValue value="1.0"/>
      </AxisValue>
    </AxisValueArray>
```

**Profile**
- [ ] in google-font profile (FAIL)
- [ ] in universal profile (WARN)
- [ ] in opentype profile (WARN)

**Format**
- [ ] in VF check
- ~in static check~

**Comment by twardoch** [comment]
https://docs.microsoft.com/en-us/typography/opentype/spec/dvaraxistag_ital says

> Valid numeric range: Values must be in the range 0 to 1.

The word “must ” indicates the hardest requirement, not a recommendation. Any user “ital” value outside the range 0–1 is invalid. If a font encodes the slant degree in the “ital” axis, that’s an invalid font.

**Comment by vv-monsalve** [comment]
> It often happens that designers assign the italic angle as a value to the Italic instance in the STAT table. So it can't be the universal profile. But it goes against recommendation of the opentype spec, and forbidden by google fonts so far.

I think this paragraph is a little bit hard to follow, to me at least. If we are requiring for the 'ital' axis value to be boolean then the first idea ("assign the italic angle as a value to the Italic instance in the STAT table") shouldn't be permitted.
In that way, this check could be included in the Universal profile and should be a FAIL.


**Profile**
- ~~in google-font profile (FAIL)~~
- [x] in universal profile (FAIL)
- ~~in opentype profile (WARN)~~

**Format**
- [x] in VF check
- ~in static check~

**Comment by RosaWagner** [comment]
> The word “must ” indicates the hardest requirement, not a recommendation. Any user “ital” value outside the range 0–1 is invalid. If a font encodes the slant degree in the “ital” axis, that’s an invalid font.

Then yes, should be a FAIL in the opentype profile.

I thought about a WARN in the universal profile cause this is common practice among designers and foundries to avoid the controversial slant axis. So many are having ital[0-12] for example, and even Glyphs recommends it in its forum. But yes it doesn't mean that it is "best" practice since it would mess up font usages the day browsers, apps and OS decide to read the open type spec more thoroughly. So I would follow @vv-monsalve and @twardoch and put it as FAIL also in Universal profile, and then see if it attracts complaints from designers who could make a better case if needed.

**Comment by twardoch** [comment]
`ital`=12 is a clear contradiction of a _hard_ requirement of the OT spec. 

If some font tool maker is recommending such a solution on some forum, then that tool maker is irresponsible in my view. 

There are only 5 registered axes, and a potential gazillion of custom ones. Anybody is free to make a font with a custom axis, e.g. `ITAL`, which can be named `Italic`, and then that axis can have any range you want. 

But registered axes should stick to the spec, and all bakery profiles should report `ital` with a range outside 0–1 as a fail. We must stop this nonsense while there is still time.
